src/sql/Expr.cpp

```diff
@@ -140,6 +140,14 @@ Expr* Expr::makeDateLiteral(char* string) {
     return e;
 }
 
+Expr* Expr::makeIntervalLiteral(int64_t duration, DatetimeField unit) {
+    Expr* e = new Expr(kExprLiteralInterval);
+    e->name = strdup("INTERVAL");
```

Comment:
I am not sure about the use of the `name` field here (and suggest removing the line (and also in `makeExtract` and `makeCast`)). I think the purpose of the `name` field was storing actual string values, which may differ among different Expression instances (e.g. different string literals) and not the name of the expression type.

Reply:
Otherwise, all changes look fine for me. Thanks for your revision.